packages/vm/src/runTx.ts

```diff
@@ -211,6 +210,12 @@ export default async function runTx(this: VM, opts: RunTxOpts): Promise<RunTxRes
       const { tx } = opts
       // Do not include sender address in access list
       const removed = [tx.getSenderAddress()]
+      // Add the active precompiles as well
+      // Note: `_precompiles` is always updated if the hardfork of `common` changes
+      const activePrecompiles = this._precompiles
```

Comment:
can we use the public accessor you added without the underscore here? and also on L267

Reply:
Done!